app/api/v1/endpoints/seasonal_forecast_list/schema.py

```diff
@@ -0,0 +1,8 @@
+from typing import Literal
+from pydantic import BaseModel
+
+country_name = Literal["zm", "mw"]
```

Comment:
nit(non-blocking)
I feel like a lot of endpoints use a country list, might be worth refactoring to some sort of shared/common model